test/Core.Test/Services/HandlebarsMailServiceTests.cs

```diff
@@ -27,6 +35,137 @@ public HandlebarsMailServiceTests()
             );
         }
 
+        [Fact(Skip = "For local development")]
+        public async Task SendAllEmails()
+        {
+            // This test is only opt in and is more for development purposes.
+            // This will send all emails to the test email address so that they can be viewed.
+            var namedParameters = new Dictionary<(string, Type), object>
```

Comment:
Very cool

Reply:
Thanks! I was to lazy to go through the process of sending the emails.